### DISCUSSION TITLE
I want a personal license for github copilot but I cannot add one because of an organization I belong to

### DISCUSSION THREAD

**Discussion by hdpinto** [community/community#40983] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm attempting to subscribe to github copilot, I use it at work and would like access to it for personal projects. When I log into my github account (same one I have used for my work accounts over the years) it displays the following:

![image](https://user-images.githubusercontent.com/1548970/206085044-6fca5215-3088-4d36-8d4d-2399134e4272.png)

I cannot use copilot on any machine that is not my work machine because it would require authenticating using my work credentials. How can I add a personal license to my own personal account so that I don't need to touch my work credentials? Is this an unsupported or unimplemented feature?

**Comment by ossanna16** [comment]
Hi there @hdpinto and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
